lte/protos/apn.proto

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2021 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+
+package magma.lte;
+option go_package = "magma/lte/cloud/go/protos";
+
+// For details about values see 29.212
+message QoSProfile {
+    int32 class_id = 1;
+    uint32 priority_level = 2;
+    bool preemption_capability = 3;
+    bool preemption_vulnerability = 4;
+}
+
+// APN authorized bitrate
+message AggregatedMaximumBitrate {
+  // Maximum uplink bitrate
+  uint32 max_bandwidth_ul = 1;
+  // Maximum downlink bitrate
+  uint32 max_bandwidth_dl = 2;
+}
```

Comment:
see the changes done in terms of units in other protos (e.g., https://github.com/magma/magma/blob/91bd361fa53cf6e079300624b9dcda2bd5f38e02/lte/protos/subscriberdb.proto#L144)

Reply:
please adopt the same...